packages/notification-service/locales/es.json

```diff
@@ -8,5 +8,7 @@
   "gold": "CELO",
   "paymentRequestedTitle": "Pago solicitado",
   "paymentRequestedBody": "Se ha solicitado pago de {{amount}} {{currency}}.",
-  "rewardReceivedTitle": "Recompensa recibida"
+  "rewardReceivedTitle": "Recompensa recibida",
+  "inviteTitle": "Tu amigo acaba de recibir tu transferencia",
+  "inviteBody": "¡La persona a la que le enviaste fondos acaba de registrarse en Valora! Ahora pueden enviarse dinero entre sí utilizando sus números de teléfono celular a tasas casi nulas."
```

Comment:
We shouldn't touch this ourself. Crowdin takes care of these too.

Reply:
Oh, that's nice I thought that was only for the wallet code.

These strings were already checked by denisse, do I still need to remove them from here or is it fine to leave them for this time?

Reply:
We should better remove, I've seen it creating merge conflicts.